src/win32_ucontext.c

```diff
@@ -71,7 +71,7 @@ void jl_makecontext(win32_ucontext_t *ucp, void (*func)(void))
     jmpbuf->Rip = (unsigned long long)func;
     jmpbuf->Rsp = (unsigned long long)stack_top;
     jmpbuf->Rbp = 0;
-    jmpbuf->Frame = 0; // SEH frame
+    jmpbuf->Frame = ~0L; // SEH frame
```

Comment:
Isn't `long` 32bit on windows?

Reply:
`L` is long long in recent standards versions, but I guess we should just spell this out as `~(uint64_t)0`.